### PR TITLE
Showing apps if query is empty, styles and dimensions

### PR DESCRIPTION
**Pull request by mtxr**
- Show apps on window open
- Config accepts Width and Height Setup
- Styles from a styles file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,4 +131,33 @@ websearch:
   # on your system. You can specify absolute path to an image file as well,
   # e.g. `/home/me/.projektor/google.png`
   icon: web-browser
+ui:
+  # use as a string. In a near future it will be proportinal to screen size (%)
+  width: "600"
+  height: "300"
```
Comment (yamnikov-oleg) [inline README.md]: Hi! Nice work :) Do I get it correctly: these values are strings to allow support of percentages in the future?

If so, could you reword the comment to something like: `These values must be quoted so they would be parsed as strings. This is required for future support of sizes proportional to screen size.`?

File: README.md [hunk 2]
```diff
@@ -131,4 +131,33 @@ websearch:
...
 ```
+
+Projektor styles should be placed at `~/.projektor/styles.css`.
+
+Here's is de default style:
```
Comment (yamnikov-oleg) [inline README.md]: This part should be spelled `Here is the default style:`

**Review by yamnikov-oleg — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mtxr) [reply README.md]: It was a typo :man_facepalming:

**Comment by mtxr** [comment]
@yamnikov-oleg fixed.

**Review by yamnikov-oleg — APPROVED** [review]

**Pull request merged**